applications/tweets/models.py

```diff
@@ -18,43 +19,9 @@ class Meta:
         ordering = ["-id"]
 
 
-class Address(models.Model):
```

Comment:
Abstract Base Classten inherit ederek bu fieldlari otomatik olarak aliyoruz. Eger makemigrations calistirirsaniz django'nun herhangi bi table'in herhangi bi field'in degistirmedigini gorebilirsiniz.